auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -254,12 +184,8 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
      */
     public Task<Void> delete(@NonNull FragmentActivity activity) {
         // Initialize SmartLock helper
-        CredentialTaskApi credentialHelper = GoogleSignInHelper.getInstance(activity);
-
-        return getDeleteTask(credentialHelper);
```

Comment:
Yeah, sorry about that. These changes were introduced when deprecating the old `delete(Activity)` method which is why I undid them here, but I can see your point of how that's confusing.